packages/libretro/melonds/package.mk

```diff
@@ -34,6 +34,10 @@ PKG_TOOLCHAIN="make"
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
+if [ "$TARGET_ARCH" = "x86_64" ]; then
```

Comment:
I guess I should remove this check, right?

Reply:
sure melonds is x86_64 only now

Reply:
@Ntemis there is a bug, melonds is x86_64 only, but it will add to build plan, thus mesa will add too, and build failed for the projects do not use mesa.

Reply:
I don' know how to fix it, thus keep the check.